src/patternfly/components/AppLauncher/app-launcher-menu-item-external-icon.hbs

```diff
@@ -1,6 +1,9 @@
-<i class="pf-c-app-launcher__menu-item-external-icon fas fa-external-link-alt{{#if app-launcher-menu-item-external-icon--modifier}} {{app-launcher-menu-item-external-icon--modifier}}{{/if}}"
-  aria-hidden="true"
+<span class="pf-c-app-launcher__menu-item-external-icon{{#if app-launcher-menu-item-external-icon--modifier}}{{app-launcher-menu-item-external-icon--modifier}}{{/if}}"
```

Comment:
need a space here in the conditional, so that if `app-launcher-menu-item-external-icon--modifier` is set, there will be a space between `pf-c-app-launcher__menu-item-external-icon` and the value of `app-launcher-menu-item-external-icon--modifier`

```suggestion
<span class="pf-c-app-launcher__menu-item-external-icon{{#if app-launcher-menu-item-external-icon--modifier}} {{app-launcher-menu-item-external-icon--modifier}}{{/if}}"
```